### PR TITLE
Blog Controllers' Tests 

### PR DESCRIPTION
**Pull request by Kccendana**
### Project requirements

- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
   - If response status was correct.
   - If a correct template was rendered.
   - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -1,17 +1,30 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    it 'returns http success' do
-      get '/users/index'
-      expect(response).to have_http_status(:success)
+RSpec.describe UsersController, type: :controller do
+  describe 'GET #index' do
```
Comment (mukhammadrizooff) [inline spec/requests/users_spec.rb:4]: - [ ] [REQUIRED] Good job so far! Spec file include tests that ensure the response body includes the correct placeholder text for `user_spec`. Example : `expect(response.body).to include("Welcome to user index page")`, please implement it 🙏🏼

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
   end
 
-  describe 'GET /show' do
-    it 'returns http success' do
-      get '/posts/show'
-      expect(response).to have_http_status(:success)
+  describe 'GET #show' do
+    it 'returns a success response' do
+      get :show, params: { user_id: user.id, id: post.id }
+      expect(response).to be_successful
```
Comment (mukhammadrizooff) [inline spec/requests/posts_spec.rb]: - [ ] [REQUIRED] Good job so far! Please make sure that for `posts show` you add tests that check 🙏🏼 

- If the response status was correct.

- If the response body includes correct placeholder text.

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Kccendana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @Kccendana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Added tests requested by the previous reviewer



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**